### DISCUSSION TITLE
Shared and Hosthome permession denied

### DISCUSSION THREAD

**Discussion by P1c1s** [Kidel/Kathara#319] [Q&A]
I have some problems with shared and hosthome directories. I enabled from settings the "Automatically mount /hosthome on startup" but doesn't work.
`ls: cannot open directory 'hosthome/': Permission denied`
Thanks advanced

**Comment by tcaiazzi** [comment]
Dear @P1c1s, 

I just tested this on my Windows host, and it works perfectly.

![immagine](https://github.com/user-attachments/assets/4050e88f-a6e1-41f8-abbc-d5a7cbf89a8e)

Could you please share the exact steps you’re following?

Thanks!

**Reply by P1c1s** [reply]
Hi @tcaiazzi, 
I'll list the steps for you:
1. I downloaded the kathara-lab_basic-ipv4
2. I started the lab with `katahra lstart`
3. On pc1 terminal emulator, I tried to save file in /shared
Anyway if I put file in shared directory from lab, when I open /shared on pc1 the directory is empty.

P.s. I tried also with others with others labs such as kathara-lab_one-bridge. Maybe It's a linux permission problem.  
![Screenshot From 2024-11-19 16-05-07](https://github.com/user-attachments/assets/b8929a45-2c2a-4954-80f6-c948cd49657c)

**Comment by tcaiazzi** [comment]
Hi @P1c1s, 

Sorry for the delay! 

On which platform are you working? (Ubuntu, Debian, Fedora...)

I guess Fedora, right? 😇 

What version of Docker and Kathará are you using? 

Thanks!

**Reply by P1c1s** [reply]
Hi @tcaiazzi,
Here all tech specs.
-  `Linux fedora 6.11.7-300.fc41.x86_64 #1 SMP PREEMPT_DYNAMIC Fri Nov  8 19:23:10 UTC 2024 x86_64 GNU/Linux`
-  `Docker version 27.3.1, build 2.fc41`

**Comment by tcaiazzi** [comment]
Dear @P1c1s, 

I just tried on my Fedora 40 system, and the `shared` directory works fine. 

I'm updating the system to Fedora 41 to have your same setting. 

I'll keep you updated 😄

**Reply by P1c1s** [reply]
Thank you so much @tcaiazzi.
